addons/website_blog/views/website_blog_components.xml

```diff
@@ -89,7 +89,7 @@
 <!-- ======   Template: Post Breadcrumbs   =====================================
 ============================================================================ -->
 <template id="post_breadcrumbs">
-    <nav aria-label="breadcrumb" t-attf-class="breadcrumb flex-nowrap py-0 px-0 css_editable_mode_hidden #{additionnal_classes or ''}">
+    <nav aria-label="breadcrumb" t-attf-class="breadcrumb flex-wrap css_editable_mode_hidden #{additionnal_classes or ''}">
```

Comment:
Is flex-wrap actually needed?

Reply:
To wrap the elements of breadcrumb in mobile display mode.

With `flex-wrap`: 
<img width="355" alt="Capture d’écran 2021-04-19 à 12 09 32" src="https://user-images.githubusercontent.com/18035234/115219839-2d7e6300-a108-11eb-942e-5ab3fcafc92a.png">

With `flex-nowrap`:
<img width="355" alt="Capture d’écran 2021-04-19 à 12 09 02" src="https://user-images.githubusercontent.com/18035234/115219938-471faa80-a108-11eb-8bd4-81eb37344ae0.png">

Reply:
And without flex-wrap like I said?

Reply:
yep my bad the `breadcrumb` class has `flex-wrap: wrap;`